eq-author/src/apollo/createApolloErrorLink.js

```diff
@@ -9,14 +9,19 @@ import { signOutUser } from "../redux/auth/actions";
 
 export const errorHandler = (getStore, error) => {
   const { networkError, graphQLErrors } = error;
+
   setSentryUser(window.localStorage.getItem("accessToken"));
-  if (networkError) {
+  if (networkError && networkError.bodyText === "User does not exist") {
+    getStore().dispatch(signOutUser());
+  } else if (
+    networkError &&
+    networkError.bodyText === "Unauthorized questionnaire access"
```

Comment:
Can we check the status code for 403 instead?

Reply:
Now checks for status code.